packages/cli/src/util/build/monorepo.ts

```diff
@@ -55,21 +55,40 @@ export async function setMonorepoDefaultSettings(
   }
 
   if (monorepoManager === 'turbo') {
-    // No ENOENT handling required here since conditional wouldn't be `true` unless `turbo.json` was found.
-    const turboJSON = JSON5.parse(
-      fs.readFileSync(join(cwd, 'turbo.json'), 'utf-8')
-    );
+    const [turboJSONBuf, packageJSONBuf] = await Promise.all([
+      fs.readFile(join(cwd, 'turbo.json')).catch(() => null),
+      fs.readFile(join(cwd, 'package.json')).catch(() => null),
+    ]);
+
+    let hasBuildPipeline = false;
+
+    if (turboJSONBuf !== null) {
+      const turboJSON = JSON5.parse(turboJSONBuf.toString('utf-8'));
+
+      if (turboJSON?.pipeline?.build) {
+        hasBuildPipeline = true;
+      }
+    } else if (packageJSONBuf !== null) {
+      const packageJSON = JSON.parse(packageJSONBuf.toString('utf-8'));
+
+      if (packageJSON?.turbo?.pipeline?.build) {
+        hasBuildPipeline = true;
+      }
+    }
 
-    if (!turboJSON?.pipeline?.build) {
+    if (!hasBuildPipeline) {
       output.warn(
-        'Missing required `build` pipeline in turbo.json. Skipping automatic setting assignment.'
+        'Missing required `build` pipeline in turbo.json or package.json Turbo configuration. Skipping automatic setting assignment.'
       );
       return;
     }
 
     setCommand(
       'buildCommand',
-      `cd ${relativeToRoot} && npx turbo run build --filter=${projectName}...`
+      `cd ${relativeToRoot} && npx turbo run build --filter={./${relative(
```

Comment:
Do we need to worry about Windows here and have `.\\`?

Reply:
Great question - i assume CI will fail if thats the case. But let me ask Turbo right now too

Reply:
https://github.com/vercel/vercel/actions/runs/3483582850/jobs/5827228329#step:8:2934 ironically CI timeout on an Nx test, but all the Turbo ones seemed to pass on Windows

Reply:
I think we should always `path.join` unless we specifically have a good reason not to. It's more likely to save us the trouble, right?

Reply:
It's kinda annoying, but `path.join()` will strip the `./`:

```
> path.join('.', 'foo')
'foo'
```